drawSvg/elements.py

```diff
@@ -444,9 +446,8 @@ def writeChildrenContent(self, idGen, isDuplicate, outputFile, dryRun):
     def appendLine(self, line, **kwargs):
         self.append(TSpan(line, **kwargs))
 
-class TSpan(DrawingBasicElement):
-    ''' A line of text within the Text element. '''
-    TAG_NAME = 'tspan'
+class TextContainingElement(DrawingBasicElement):
+    ''' A private class used for elements that have content to be written. '''
```

Comment:
Suggested clarification:
```suggestion
    ''' A private parent class used for elements that only have plain text content. '''
```